interface/client/templates/layout/webviews.js

```diff
@@ -11,13 +11,13 @@ The main section template
 @constructor
 */
 
-Template['layout_webviews'].helpers({
+Template.layout_webviews.helpers({
```

Comment:
The way i wrote it, is for the reason of readability. So i want to keep them, as `Template['layout_webviews']`